src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```
@@ -329,22 +326,15 @@ private bool VersionEquals(NuGetVersion v1, NuGetVersion v2)
             return v1.Equals(v2, VersionComparison.Default);
         }
 
-        private long? _downloadCount;
-
         public long? DownloadCount
         {
             get
             {
-                return _downloadCount;
-            }
-            set
-            {
-                _downloadCount = value;
-                OnPropertyChanged(nameof(DownloadCount));
+                return (_packageModel as RemotePackageModel)?.DownloadCount ?? 0;
```

This should be null if there is no download count so it does not show up. A default of 0 is misleading if the download count is unknown.